CHANGELOG.md

```diff
@@ -3,8 +3,8 @@
 ### Features
 
 * `ilab` now uses dedicated directories for storing config and data files. On Linux, these
-  will generally be the XDG directories: `~/.config/instructlab` for config and
-  `~/.local/share/instructlab` for data. On MacOS, both the config and data is
+  will generally be the XDG directories: `~/.config/instructlab` for config,
+  `~/.local/share/instructlab` for data, and `~/.cache` for temporary files, including downloaded models. On MacOS, both the config and data is
```

Comment:
nit: this document seems to be (mostly) wrapped at 100 characters. Perhaps you could wrap it accordingly here too.

Reply:
Done!